custom_components/tuya_local/devices/silvercrest_coffee_maker.yaml

```diff
@@ -0,0 +1,51 @@
+name: Silvercrest Coffee Maker
+products:
+  - id: iegq4tjqeuiew5cl
+    name: Coffee Maker
+primary_entity:
+  entity: water_heater
+  icon: "mdi:coffee-maker"
+  dps:
+    - id: 1
+      type: boolean
+      name: operation_mode
+      mapping:
+        - dps_val: false
+          value: "Off"
+        - dps_val: true
+          value: "On"
+    # Dummy dps, water_heater requires a temperature.
+    # The coffee maker has no sensor for this so we put in a fake dps for it.
+    - id: 8000
+      type: integer
+      name: temperature
+      unit: C
+      optional: true
+      range:
+        min: 0
+        max: 0
+    - id: 4
+      type: string
+      name: current_state
+    - id: 12
+      type: string
+      name: keep_warm_min
+    - id: 101
```

Comment:
Does not need to be here if it is unused by the water_heater, and has a dedicated entity below.